22nd-jan/cacheFunction.js

```diff
@@ -0,0 +1,24 @@
+Utils = {
```

Comment:
This is polluting the global object. Utils might be a functionality already a part of global and this might conflict with that.